packages/vocabulary/src/styles/header.scss

```
@@ -63,7 +63,8 @@ $navbar-breakpoint: $tablet;
     }
 
     & > .navbar-item {
-      margin-left: $space-large;
+	margin-left: $space-large;
+	color: $color-dark-gray;
```

```suggestion
      margin-left: $space-large;
      color: $color-dark-gray;
```

This looks great, it just needs to be spaces instead of tabs.

---

@zackkrida Shall I implement this change.But I have to open a new PR since I can't pass commits to the PR.

---

We can wait for the original contributor to revise their work, thank you!

---

Done